File: website/public/js/engine/entities/spell/SpellScaling.js
/**
 * @constructor
 * @param {Array} serialized
 */
function SpellScaling(serialized) {
	this.castTimeStart = serialized[0];
	this.castTimeEnd = serialized[1];
	this.intervals = serialized[2];
	this.distribution = serialized[3];
}
SpellScaling.prototype = {
	castTimeStart : 0,
	castTimeEnd : 0,
	intervals : 0,
	distribution : 0
};

File: tools/cjt/src/org/chardev/cjt/entity/SpellScaling.java
package org.chardev.cjt.entity;

public class SpellScaling {
	protected final int castTimeStart;
	protected final int castTimeEnd;
	protected final int intervals;
	protected final int distribution;
	
	public SpellScaling( int castTimeStart, int castTimeEnd, int intervals, int distribution ) {
		this.intervals = intervals;
		this.castTimeEnd = castTimeEnd;
		this.castTimeStart = castTimeStart;
		this.distribution = distribution;
	}

	public int getCastTimeStart() {
		return castTimeStart;
	}

	public int getCastTimeEnd() {
		return castTimeEnd;
	}

	public int getIntervals() {
		return intervals;
	}

	public int getDistribution() {
		return distribution;
	}
}
